### PR TITLE
Adds resetYourPassword to Schema

### PR DESCRIPTION
**Pull request by abhitip**
Current schema - https://app.segment.com/artsy-engineering/sources/force-production/schema/events/ev_1ZJKQD2cwDGCUUcHVcYVC4rrAQs

I'm proposing removing 5 properties from the current schema:
- `trigger_seconds`
- `action` - this is just the event name, not sure why it's populating in the segment schema
- `modal_copy`
- `type` - this is just "forgot" for all of these events
- `service` - I think this will always be "email" because if the user forgets a facebook or apple password they'll have to go to that service's website to request a password reset.

### CODE REVIEW

File: src/Schema/Event.ts [hunk 1]
```diff
@@ -68,6 +68,30 @@ export interface CreatedAccount {
...
+ *    trigger: "timed"
+ *    auth_redirect: "https://artsy.net/artist/andy-warhol"
+ *  }
+ * ```
+ */
+export interface ResetYourPassword {
+  action: ActionType.resetYourPassword
+  context_module: AuthContextModule
+  intent: AuthIntent
+  trigger: AuthTrigger
```
Comment (eessex) [inline src/Schema/Event.ts:91]: trivial, not alphabetized

Comment (abhitip) [reply src/Schema/Event.ts:92]: fixed this and the other ones too!

**Comment by abhitip** [comment]
Spoke to @anipetrov and decided that we should keep `authImpression`-identifying properties (modal_copy, trigger_seconds, type, onboarding)  in the other events - `resetYourPassword`, `createdAccount`, and `successfullyLoggedIn` - for analysis purposes. It will be easier to analyze the other 3 events if they have the properties already on them, rather than having to associate them back to the auth impression event that preceded them. I also added `user_id` to `createdAccount` and `successfullyLoggedIn`.

**Comment by eessex** [comment]
Somehow I messed up the process of pushing to this branch and accidentally published the changes @abhitip made into master already. Very sorry, this now shows some unrelated changes of mine. 
Original schema updates are merged and visible here: https://github.com/artsy/cohesion/blob/master/src/Schema/Event.ts#L101

**Comment by eessex** [comment]
I am closing this PR for the moment to move the current changes elsewhere.

**Pull request closed without merging**